app/src/main/java/net/squanchy/search/SearchActivity.kt

```diff
@@ -123,9 +120,8 @@ class SearchActivity : AppCompatActivity(), SearchRecyclerView.OnSearchResultCli
         } else {
             emptyView.isInvisible = true
             searchRecyclerView.isVisible = true
-
-            searchRecyclerView.updateWith(searchResult, this)
         }
+        searchRecyclerView.updateWith(searchResult, this)
```

Comment:
we now update the recyclerview even when there are no search result. This is done for consistency, since we now have animations for adapter changes. Otherwise the next search result would animate from the state x to the state x + 2, skipping x + 1 (empty adapter)